src/main/java/org/springframework/social/salesforce/api/CommunityOperations.java

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (C) 2019 https://github.com/jottley/spring-social-salesforce
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.social.salesforce.api;
+
+import java.util.List;
+
+public interface CommunityOperations {
```

Comment:
To keep consistency with how the operations are named this should be named ConnectOperations.  The naming convention follows the top most value after the API version number as the name.  This change will cascade down to a few other places as well.